Lib/test/test_pdb.py

```diff
@@ -1110,6 +1112,44 @@ def test_readrc_kwarg(self):
             if save_home is not None:
                 os.environ['HOME'] = save_home
 
+    def test_find_function_with_invalid_filename(self):
+        self.assertIsNone(pdb.find_function('foo', 'invalid filename'))
+
+    def test_getsourcelines(self):
+        def foo():
+            pass  # pragma: no cover
+        lines, lineno = inspect.findsource(foo)
+        expected_lines = inspect.getblock(lines[lineno:])
+        expected_lineno = lineno + 1
+        actual_lines, actual_lineno = pdb.getsourcelines(foo)
+        self.assertEqual(actual_lines, expected_lines)
+        self.assertEqual(actual_lineno, expected_lineno)
+
+    def test_getsourcelines_with_module_frame_obj(self):
+        for frame, *unused in inspect.getouterframes(inspect.currentframe()):
```

Comment:
Yes,  I would like it to be tested. Initially, when I've seen test coverage of stdlib I wanted to write some tests to improve it. Now it seems like tough task, because it is not even clear who should approve my PRs. If I address all comments here and in bpo, whom can I ask to approve my PR?